.gitignore

```diff
@@ -1,6 +1,7 @@
 ccpi_viewer.egg-info
 *.pyc
 __pycache__
+*.log
```

Comment:
is this necessary?

Reply:
well, the viewer often makes log files which we don't want to commit